src/task.c

```diff
@@ -235,6 +235,52 @@ JL_DLLEXPORT void *jl_task_stack_buffer(jl_task_t *task, size_t *size, int *tid)
     return (void *)((char *)task->stkbuf + off);
 }
 
+JL_DLLEXPORT void jl_active_task_stack(jl_task_t *task, char **start, char **end)
+{
+    if (!task->started) {
+        *start = *end = 0;
+        return;
+    }
+
+    int16_t tid = task->tid;
+    jl_ptls_t ptls2 = (tid != -1) ? jl_all_tls_states[tid] : 0;
+
+    if (task->copy_stack && ptls2 && task == ptls2->current_task) {
```

Comment:
This code is strongly inspired by `gc_scrub_task` with a few differences:
- we adjust for `ROOT_TASK_STACK_ADJUSTMENT`
- we adjust the stack end if `task->copy_stack > 1` and `COPY_STACKS` is #defined.